doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/community-supported-cluster-manager/index.rst

```diff
@@ -9,6 +9,13 @@ Community Supported Cluster Managers
 
     If you're using AWS, Azure or GCP you can use the :ref:`Ray Cluster Launcher <cluster-cloud>` to simplify the cluster setup process.
 
+
+To use Ray autoscaling on other Cloud providers or cluster managers, you can implement the ``NodeProvider`` interface (100 LOC) 
+and register it in `node_provider.py <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/node_provider.py>`_. Contributions are welcome!
```

Comment:
I'm not sure what `register` means here.
The way to do is to specify an "external" node provider by using this in the cluster config
```
provider:
  type: "external"
  module: "my.module.MyCustomNodeProvider"
```
I'm not sure if this is in the current docs.